index.html

```diff
@@ -0,0 +1,103 @@
sjXIvQkNXuvdudbHOT4zCJ9cWZStI7mRFfD+amp1+5lTOTiS1VeWEmPTP+qL8DBcrdmXRWK8VdbhQFP8wxqLqXPeU1HItBklp5UEvTdiqrA7RWmjV4NSRLwnZCrze1ONIOusQx+1yxfh9GqaKEUdtG2E7oeOYzGdRXdbAzF0siOeN6ZMyWOitCzPx4irH0IgKN9fhhjkV2uLGL1kYNx8IHYidqCZGo2GWghldRZdD3FMv6UWjp5yvzTY4ZHtY3MnmocmP3gPPwUrZzYfqlmK8Jy3wUFqQUjafti8Vr27T71RwwthsZne3sH+a4ISzSC2tEHccD4NhwvIm04kCztiBIUnnbwTn5Kl8ogsy2VDDwCxw3qtwtUZdCVQaPF56jUkqNJnE3KAHNeoeZNQWc7WhCgfBPsVVBG/7UWXp+8/dPcyzccGiru+g6T2s1JFHHsdwKlGjaJmzpo02WF4EatOqAxExa8rbwMl7MiLnA8S7TZ8oamk1lykbEJIrzX/mQAQmOg54MwsLdik+UCi8xKob6iBAForvGGfw0xzK2aOy2a6VVaHExmvBJWMfxSMbVC/X6XKRIaXEWYJ5k7yCqVattLHzchsOwMEyzWo6fX+atj+F9pKs1ut++J4WPPhQeDOuCGvCRepHJeZi4LBSk9tyAOkIw61yWlKKnotId5+c3qf40xzL6GMRfiiZA47UIxxFZsfrUcSwcGaVuUsjL81xeVvvxBc6reuTiUw+iiFQdhDTn6ibd6+KthuOH/gpHIcEI6WuP9NLR1zyCyRjfqCtM+zXQNocwy4BoH2bGYjJilaBxQ6z6n+mY544Z5kUMW+rzdN/b+SzrH+d4Lh0CYbeAxz6VXJ87vZcMjhVLBJETHyXtpkzb05sogncglY0JK84rQbglhtygmQYkHKvzqYbj9ko47tJHUX6wFgOZeelEtyhtLlftN4Ki1Vq2jyrBhkOc2UpsZDbtgyG424taXmxzsmZL8egRejundP04x40vSTJOj4diWaw/iZqjyIrVcqxkQlY5e+v5fL0I5BDBK91OmBkBgVC/LTOtgmjVWpbfGmSne5bZTJqOHHpDVstkvCAO95tpVjbnaSWDylysys0GOPkaVUlAvdEHtbFQJFkvlDkpwAZntcayI3yQ0UAaMKYJTqyvDYf6gNnQ59yKP8+xCHCzFqdV7qzCAWgctRxfyL1NDPukracyWyqTkqyNkrSkIpLJswm6g4/XcdzWMomTqGqO3BwnEz9WcmCWVcY5kq3G6Q3X6qkHw0yeYBGNkvnOGzN+C1cLkkdIHIqMDmB63Dm74ec55jtoayCOAKzn+Fs59BILzVscW1bE+EwOPbqFpa9W55rfBvWQKv5O3yahuz3TfUe22czf6bHWpm1m/yeW+sRO+4kK1q/YDMjRaOzZfci/gWM1bd1EeIQanuG40by8F0YiUws7EmcaH7XhXSS7pt52eqpvDVI2K2X1e2FCcEONnfnQTGXAN+yVue2yojkD6I3V0NoARVOeknPG0+/guCS5JjYfHYXaeY7jxunSnjYVK1WSO7LIBo811UGKpXrGPh7XjRGM1dmT5TW7w8IhKETU9kfWNn3WE4wIp5+fThomgSBXWGKmh7Ic+C0cl5aOKxSDorFU/85y3FgTR4cR4hytG6mQGTqnKIFdcyy+Vyg65wM5ubYHlf16X+h9c3TtTVW3wA6r+RiYJKn7GEK3m/9DrEfWqQLcp30uter3cNzYfRJrj7m+q/s8x8495n23x24n51mdnejYY07wh6qLnvVzzbtmZcpy8YcZPh3Ye8xRslXG5Qo5hfFcbjsYOjtD5hA7Rgh1AZw9M2IIhy7oXvteDgcu6E1+4Cc8hLDfEs6KCEPj44V1VkRTc0Cf+CEVYVqTxDAaEtljKCTD2rN8jxHUwU5j5OjM3tSzInAU3BRaATgrIkxdvTVYd8vKKM8n6Fg4yu0ec73R25Pa6OoEENc2NREey+uC/eCXd8hk5tI+l3+7aXLoC1n7xC8/6b32/A7X+ZwqnuAFjjNf+DEpYYzzT/OUlbl4sDZRpVhsccwOm9kuivo2ZOJVpzOuAPG2kAT9g9mZg427wfIjzbcATqAJhps6W6W3vuGNJjgfGwfdUB4dW1cgIhvWDuNGi8311M5o7l12dP1WVMcdzWbT4l//jMmumP6n5+23jabFy788MY61ZlRMzj/fmZTFRtPMkqp1HIv00TOxfj7VHbGVP82xhwY3xwO5d5nUZSyLyJZtPnuO/ypsnevxUXX0uiVWJtVKS7Gm6/G1jjn2uISuQ6/ufCRcY66AnYr1jlo5XOczYjJcr77iL4h4nAOLkWkm14CfSIO57RY6JmT7yA6F2LJYc6wnT9D83fBkP+bxJ/Bk+TLbX4xi8jbgoav03Xxq8MXCZPMJn+z6T8ewTPur/UiMx3lYcadenxFLXntiXU4Nraq9Z1sjmpXXk5Gs2fA07pRcylfzuBLowVtKqs6Eu7jJqrKz2vz4p2ivrti7N5b8QRXnJfe8R/IYDnrkgJXg6PHHQHUnESDecMYSHrTgv+IVxLl0pK+ZExacnBNehIizUmgSg1e5/h602CEtbHcWnEGbinkL8UkUsmyzTtFnYQ4icgEfXkNe5IOK5yLQho3HHwff37RtzU5DHsxCqpKVwWl9BHc/3z9WvEzYlaK4t4ff8AuOm9GC7WPt1/qDPa4OdponCiPxI7dYi5dOXkXqMRGRu2SvJXc+QQgTJzwLyk/jvwmdvRZYR+FboRd4tg5IxuTJmKUz40xxZ+aExx/Ds3LiYEDQux0oOwRKTmCAklc7VWu3D2U8G0Ve4frLMFgkjEOckE+nVbt7xJCSgJOg5QxyFf2Q2cooDPzv//x9mCyC6sCf/qE2YrrbdG/TCsdZbRhzeXqrzg1atX75x3o8roPniyHslyy7GDVvX/H36j08PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8fgz/AMBpcwBl6mK+AAAAAElFTkSuQmCC">
```

Comment:
It looks like you are using an image from the internet.
It´s a good practice to add the image resources on your repo and avoid using links from the internet
- [ ] Kindly Add the image resource like downloaded resource on your repo.